src/graph/entities/graph_entity.c

```diff
@@ -232,7 +232,7 @@ inline int GraphEntity_ClearAttributes
 
 	int count = ATTRIBUTE_SET_COUNT(*e->attributes);
 	
-	AttributeSet_Clear(e->attributes);
+	AttributeSet_Free(e->attributes);
```

Comment:
Please note for queries such as:
`UPDATE n = {v:1,...}` we will free and allocate, instead of just reallocating twice which.

Reply:
yes but I think we also need to handle 1 relocation when
`set n.a = 1, n.b = 2, n.c = 3`